### DISCUSSION TITLE
Bad request

### DISCUSSION THREAD

**Discussion by Rahmanay** [Budibase/budibase#4702] [Help]
Hi,

Does anyone know why users would be getting a "Bad request" notification, when submitting forms etc?

**Comment by mjashanks** [comment]
Happy to help if you could give some more info....

- open up dev tools on your browser
- go to the network tab
- submit your form
- lookout for failed requests
- paste details of them in here - especially whats in the Response tab

Thanks!

**Reply by sirio2013** [reply]
Ciao @mjashanks .
I have the same issue.
I have pasted below the online 400 status I get when I save.
Can you help me?

Basically it tells me that: "Cod. Art. Supplier":["can't be blank"],"Cod. Art. Merchant":["can't be blank"]}}
Well of course, but I am not leaving the fields blank; those fields are required.

**Response tab**
{"message":"Bad Request","status":400,"validationErrors":{"Cod. Art. Supplier":["can't be blank"],"Cod. Art. Merchant":["can't be blank"]}}

**Headers**
Request URL: https://santamadonna.budibase.app/api/ta_013da7d077a143499c44107fe767f541/rows
Request Method: POST
Status Code: 400 
Remote Address: 3.248.154.238:443
Referrer Policy: strict-origin-when-cross-origin
content-length: 139
content-security-policy: default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://cdn.budi.live https://js.intercomcdn.com https://widget.intercom.io; style-src 'self' 'unsafe-inline' https://cdn.jsdelivr.net https://fonts.googleapis.com https://rsms.me https://maxcdn.bootstrapcdn.com; object-src 'none'; base-uri 'self'; connect-src 'self' https://api-iam.intercom.io https://api-iam.intercom.io  https://api-ping.intercom.io https://app.posthog.com wss://nexus-websocket-a.intercom.io  wss://nexus-websocket-b.intercom.io https://nexus-websocket-a.intercom.io https://nexus-websocket-b.intercom.io  https://uploads.intercomcdn.com  https://uploads.intercomusercontent.com; font-src 'self' data: https://cdn.jsdelivr.net https://fonts.gstatic.com https://rsms.me https://maxcdn.bootstrapcdn.com  https://js.intercomcdn.com  https://fonts.intercomcdn.com; frame-src 'self' https:; img-src http: https: data: blob:; manifest-src 'self'; media-src 'self' https://js.intercomcdn.com; worker-src 'none';
content-type: application/json; charset=utf-8
date: Tue, 10 May 2022 14:24:10 GMT
server: nginx
vary: Accept-Encoding
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
x-xss-protection: 1; mode=block
:authority: santamadonna.budibase.app
:method: POST
:path: /api/ta_013da7d077a143499c44107fe767f541/rows
:scheme: https
accept: application/json
accept-encoding: gzip, deflate, br
accept-language: it-IT,it;q=0.9,en-US;q=0.8,en;q=0.7
content-length: 462
content-type: application/json
cookie: _ga=GA1.1.1563283757.1651401733; budibase:auth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJob3N0aW5nIjoiY2xvdWQiLCJidWRpYmFzZVVzZXJJZCI6InVzXzNiOGYyZDJhM2U5ZTRjMWZiZDBkMDM0ZjZjMGJmNGIzIiwidGVuYW50SWQiOiJzYW50YW1hZG9ubmEiLCJ2ZXJpZmllZCI6dHJ1ZSwic2Vzc2lvbklkIjoiZTJjY2EzMzk5NTc0NGMzN2EyMDgwNzhjNzBkZjBjMmEiLCJ1c2VySWQiOiJ1c18zYjhmMmQyYTNlOWU0YzFmYmQwZDAzNGY2YzBiZjRiMyIsImlhdCI6MTY1MjE4OTc0M30.q1i0KvCitDsviuO4FWX1Yt7EcvSgld-LozET8JKR2Rw; budibase:currentapp=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhcHBJZCI6ImFwcF9kZXZfc2FudGFtYWRvbm5hX2U4NGQyM2I5OWY5NTQwODFiNDcxZjZlY2MyYWNjOTM1IiwiaWF0IjoxNjUyMTg5OTk5fQ.ZQ2DGVEZzr4CTWDhLxiCkkWJrfNd6KiX6NSEZGNy8xw; budibase:init=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2NTIxOTAwMDB9.furuoz3V060e1KqWR8bqylWnGOgMQF2YA4eEKHAOnak; budibase:init.sig=I30G_vlyjwth5Xr_egq5KhAh-2A; _ga_2JVG0FWDDG=GS1.1.1652192342.7.1.1652192350.0; ph_phc_yGOn4i7jWKaCTapdGR6lfA4AvmuEQ2ijn5zAVSFYPlS_posthog=%7B%22distinct_id%22%3A%20%22us_3b8f2d2a3e9e4c1fbd0d034f6c0bf4b3%22%2C%22%24device_id%22%3A%20%221807f36b4f85c5-002057cd0f9fe4-6b3e555b-1fa400-1807f36b4f952f%22%2C%22%24session_recording_enabled%22%3A%20false%2C%22%24active_feature_flags%22%3A%20%5B%5D%2C%22%24enabled_feature_flags%22%3A%20%7B%7D%2C%22%24initial_referrer%22%3A%20%22https%3A%2F%2Faccounts.google.com%2F%22%2C%22%24initial_referring_domain%22%3A%20%22accounts.google.com%22%2C%22%24referrer%22%3A%20%22https%3A%2F%2Fsantamadonna.budibase.app%2F%22%2C%22%24referring_domain%22%3A%20%22santamadonna.budibase.app%22%2C%22%24search_engine%22%3A%20%22google%22%2C%22%24user_id%22%3A%20%22us_3b8f2d2a3e9e4c1fbd0d034f6c0bf4b3%22%7D; intercom-session-qz2sxfuv=U1d2aTd6c1Z1YlhYY0plNE5vemtYcUVKd1o0T2JiUDBtUWdpMUcwYTdZQndEUWxCMitTSGlYV2RFNTBPK003dS0tZ1c4U1M5N2xRZFZFRWg4NFhVQ1FQdz09--8132aed64691f7e380d31923933e51257cf5d619
dnt: 1
origin: https://santamadonna.budibase.app
referer: https://santamadonna.budibase.app/app_dev_santamadonna_e84d23b99f954081b471f6ecc2acc935
sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="101", "Google Chrome";v="101"
sec-ch-ua-mobile: ?0
sec-ch-ua-platform: "Windows"
sec-fetch-dest: empty
sec-fetch-mode: cors
sec-fetch-site: same-origin
user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.54 Safari/537.36
x-budibase-api-version: 1
x-budibase-app-id: app_dev_santamadonna_e84d23b99f954081b471f6ecc2acc935
x-budibase-role: PUBLIC
x-budibase-type: client

**Reply by melohagan** [reply]
Hey @sirio2013 I would make sure the field names are definitely correct. Rather than freely type them, you should be able to select them from the dropdown e.g.
![Screenshot 2022-05-16 at 15 24 08](https://user-images.githubusercontent.com/101575380/168615043-c71ecc96-c803-47a4-96fe-b6e35f36ef4a.png)
